utils/usbmuxd/Makefile

```diff
@@ -31,7 +31,7 @@ define Package/usbmuxd
   SUBMENU:=libimobiledevice
   TITLE:=USB multiplexing daemon
   URL:=https://www.libimobiledevice.org/
-  DEPENDS:=+librt +libusb-1.0 +libusbmuxd +libopenssl +libimobiledevice
+  DEPENDS:=+librt +libusb-1.0 +libusbmuxd +libopenssl +libimobiledevice +usbutils
```

Comment:
slightly unrelated but librt should be removed. Both musl and glibc now implement librt as a stub with the functions being directly in libc.

Reply:
Thanks, I removed `librt`.